ElasticBox/user-authentication.md

```diff
@@ -71,9 +115,21 @@ LDAP groups get automatic access to team workspaces in ElasticBox when you enabl
 This gives your developers, operations engineers, or IT admins access to the same deployment assets to do their part in automating with necessary access levels. Follow these steps to sync with LDAP groups.
 
 **Steps**
+<<<<<<< HEAD
```

Comment:
Clean up this section. remove the `<<<<<<< HEAD`, `=======`, and `>>>>>>> CenturyLinkCloud/master` lines. Also remove the duplicate content.